File: game-server/src/main/java/org/forfun/mmorpg/game/player/message/ReqPlayerLogin.java
package org.forfun.mmorpg.game.player.message;

import jforgame.socket.share.annotation.MessageMeta;
import jforgame.socket.share.message.Message;
import org.forfun.mmorpg.game.player.service.PlayerService;

@MessageMeta(cmd = PlayerService.CMD_REQ_PLAYER_LOGIN)
public class ReqPlayerLogin implements Message {

	private long playerId;

	public long getPlayerId() {
		return playerId;
	}

	public void setPlayerId(long playerId) {
		this.playerId = playerId;
	}

}

File: game-server/src/main/java/org/forfun/mmorpg/game/player/message/ReqPlayerLogin.kt
package org.forfun.mmorpg.game.player.message

import jforgame.socket.share.annotation.MessageMeta
import jforgame.socket.share.message.Message
import org.forfun.mmorpg.game.player.service.PlayerService

@MessageMeta(cmd = PlayerService.CMD_REQ_PLAYER_LOGIN)
class ReqPlayerLogin : Message {

    var playerId: Long = 0
}
